applications/IgaApplication/tests/linear_beam_thick_p4_nCP5/linear_beam_thick_p4_nCP5_test_results.json

```diff
@@ -0,0 +1,388 @@
+{
+    "TIME": [0.1],
+    "NODE_1": {
+        "DISPLACEMENT_X": [0.0],
+        "DISPLACEMENT_Y": [0.0],
+        "DISPLACEMENT_Z": [0.0],
+        "ROTATION_X": [0.0025000000000000096],
+        "ROTATION_Y": [1.1263505319408784e-16],
+        "ROTATION_Z": [0.0]
```

Comment:
this wolud also reduce pollution here

Reply:
@tteschemacher why do you write this out?
=> you can specify the components X and Y then Z won't appear in the file